src/store/model/rosman-melb.ts

```diff
@@ -0,0 +1,145 @@
+import {Store} from './store';
+
+export const RosmanMelb: Store = {
+	backoffStatusCodes: [403, 429],
+	currency: '$',
+	labels: {
+		inStock: {
+			container:
+				'#Availability > div > table > tbody > tr:nth-child(2) > td:nth-child(1)',
+			text: [
+				'1',
+				'2',
+				'3',
+				'4',
+				'5',
+				'6',
+				'7',
+				'8',
+				'9',
+				'10',
+				'11',
+				'12',
+				'13',
+				'14'
+			]
```

Comment:
This seems like this could be trouble. Perhaps just go 1-9, since it's a string and the check does a `contains`.

Reply:
included with build d33c279